recipes/sml/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, tools
+from conans.tools import check_min_cppstd
+import os
+
+
```

Comment:
```suggestion

required_conan_version = ">=1.32.0"


```

But honestly `validate()` is overkill here, you should put cppstd checks in `configure()`

Reply:
Isn't `configure()` going to be deprecated soon?

Reply:
Not to my knowledge

Reply:
No, it won't. But `validate` is the new way for validation. CCI considers all users are running the latest Conan version too.